README.rst

```diff
@@ -70,9 +57,14 @@ package. ``pip3 install wheel``
 Multiple instances
 ------------------
 Devicehub can run as a single inventory or with multiple inventories,
-each inventory being an instance of the ``devicehub``. To execute
-one instance, use the ``flask`` command, to execute multiple instances
-use the ``dh`` command. The ``dh`` command is like ``flask``, but
```

Comment:
Does removing that lines mean that you cannot run devicehub-teal on single-inventory mode?

Reply:
If I understand well, you can run DH using _'dh'_ instead of _'flask_' to run a single inventory.
_'dh run'_ 
We can close this pull and related [issue](https://github.com/eReuse/devicehub-teal/issues/25)?

Reply:
Jordi is correct. We just jus the `dh` command to initialize the db.
@slamora  as an implementation agnostic user, can you run the instructions on the README and confirm if they lead to a running deployment?

Gracias